app/controllers/candidate_interface/course_choices_controller.rb

```diff
@@ -61,10 +61,10 @@ def pick_course
         application_form: current_application,
       )
 
-      if @pick_course.other?
-        redirect_to candidate_interface_course_choices_on_ucas_path
-      elsif !@pick_course.valid?
+      if !@pick_course.valid?
```

Comment:
Could a selected course be valid but not applyable here? (I couldn't see a validation that checks that a course is `open_on_apply`).

Reply:
Yes. If a course is valid, it will fail this conditional and move to the next one that checks if it's `applyable?`. I think that's fine, right?